Dockerfile

```diff
@@ -48,6 +49,9 @@ RUN SWIFT_ARCHIVE_NAME=swift-$SWIFT_VERSION-$SWIFT_PLATFORM && \
 ENV PATH /usr/bin:$PATH
 
 # -- CUT END --
+RUN chmod 755 -R /usr/lib/swift
```

Comment:
These two lines should be after CUT END, otherwise it won't be included in main image.
Also, note in comment that this is a fix for 12/04 build and it most likely won't be needed in newer ones.